packages/components/src/List/Toolbar/ColumnChooser/Content/DefaultColumnChooser.components.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import theme from './ColumnChooser.scss';
+import ActionButton from '../../../../Actions/ActionButton';
+import ColumnDisplayer from '../ColumnDisplayer';
+
+export const DefaultHeader = ({ t }) => (
+	<div className={classNames(theme['tc-column-chooser-header'], 'tc-column-chooser-header')}>
+		{t('COLUMN_CHOOSER_HEADER_TITLE', {
+			defaultValue: 'Modify columns position',
+		})}
+	</div>
+);
+
+DefaultHeader.propTypes = {
+	t: PropTypes.func.isRequired,
+};
+
+export const getColumnDisplay = (length, onChangeVisibility, onBlurOrder, onKeyPressOrder, t) => (
+	column,
+	index,
+) => {
+	const displayerProps = {
+		...column,
+		length,
+		onBlurOrder: onBlurOrder(index),
+		onChangeVisibility: onChangeVisibility(index),
+		onKeyPressOrder: onKeyPressOrder(index),
+		t,
+	};
+	return <ColumnDisplayer key={column.label} {...displayerProps} />;
+};
+
+export const DefaultBody = ({ columns, onChangeVisibility, onBlurOrder, onKeyPressOrder, t }) => (
+	<div
+		id="column-chooser-content"
+		className={classNames(theme['tc-column-chooser-body'], 'tc-column-chooser-body')}
+	>
+		{columns.map(
+			getColumnDisplay(columns.length, onChangeVisibility, onBlurOrder, onKeyPressOrder, t),
+		)}
+	</div>
+);
+
+DefaultBody.propTypes = {
+	columns: PropTypes.array.isRequired,
+	onBlurOrder: PropTypes.func.isRequired,
+	onChangeVisibility: PropTypes.func.isRequired,
+	onKeyPressOrder: PropTypes.func.isRequired,
+	t: PropTypes.func.isRequired,
+};
+
+export const DefaultFooter = ({ onClose, onSelectAll, selectAllValue, submit, t }) => {
+	const onSubmit = event => {
+		submit(event);
+		onClose();
+	};
+	return (
+		<div className={classNames(theme['tc-column-chooser-footer'], 'tc-column-chooser-footer')}>
+			<span
+				className={classNames(
+					theme['tc-column-chooser-footer-select-all'],
+					'tc-column-chooser-footer-select-all',
+				)}
+			>
+				<span>
+					<input
+						id="select-all-checkbox"
+						name="selectAll"
+						aria-label="select all"
+						onChange={event => onSelectAll(event, !selectAllValue)}
+						type="checkbox"
+						checked={selectAllValue}
+						value={selectAllValue}
+					/>
+				</span>
+				<label id="select-all-label" htmlFor="selectAll">
+					Select All
+				</label>
```

Comment:
The markup is not correct.
You should not pass an aria label to the input, the label should be properly linked to the input via htmlFor.
But it seems that it's not the case. You should follow the structure 
```
<label>
    <input />
    <span />
</label>
```